### PR TITLE
Centralized Style Sheet formatting and standardized login screen for web and mobile

### PR DESCRIPTION
**Pull request by Larry-Langman**

### CODE REVIEW

File: front-end/CopyPasta/app/LoginScreen.js [hunk 1]
```diff
@@ -1,48 +1,77 @@
...
   },
   containerHeader: {
+    flex: .2,
     fontSize:48,
+    color: "#fff",
   },
   containerSubHeader: {
-    fontSize:24
+    flex: .25,
+    fontSize:22,
```
Comment (vb1262) [inline front-end/CopyPasta/app/LoginScreen.js:51]: why does a subheader have smaller font than the text input?

File: front-end/CopyPasta/app/LoginScreen.js [hunk 2]
```diff
@@ -1,48 +1,77 @@
...
+    backgroundColor: "#fff",
+    borderRadius: 10,
+    padding: 10,
   },
+  buttonText: {
+    color: "#01003b",
+    fontSize: 24,
+  },
+
 });
```
Comment (vb1262) [inline front-end/CopyPasta/app/LoginScreen.js:76]: Overall, seems good to go , let me know if you want me to look at anything specific

**Review by vb1262 — APPROVED** [review]

File: front-end/CopyPasta/app/LoginScreen.js (hunk 1 above)
Comment (Larry-Langman) [reply front-end/CopyPasta/app/LoginScreen.js]: you want the code to be larger font and to stand out more

File: front-end/CopyPasta/app/LoginScreen.js (hunk 2 above)
Comment (Larry-Langman) [reply front-end/CopyPasta/app/LoginScreen.js]: nope, just need the pull request to be accepted so we can transition between pages.

**Pull request merged**